pkg/cmd/build/build.go

```diff
@@ -210,7 +209,7 @@ func validateImage(imageTag string) error {
 	}
 	if (reg.IsOktetoRegistry(imageTag)) && strings.Count(imageTag, "/") != 1 {
 		prefix := constants.DevRegistry
-		if registry.IsGlobalRegistry(imageTag) {
+		if strings.HasPrefix(imageTag, constants.GlobalRegistry) {
```

Comment:
what happend if user use expanded notation to point global reg?

Reply:
I moved it from [here](https://github.com/okteto/okteto/blob/21bda07cdc4a8dcfe612a31c5ad90e9af6780650/pkg/registry/registry.go#L214) so it was an issue that we were having, but you are right. We should check the expanded form of it. Done in https://github.com/okteto/okteto/pull/3435/commits/07249fbe66556fc9812c21a0ce367208bd7d7609